https://bugzilla.mozilla.org/show_bug.cgi?id=1908317

One case was wrongly backported into 128 SRU branch and still needs to be patched.

--- firefox-128.1.0/modules/libpref/init/StaticPrefList.yaml
+++ firefox-128.1.0/modules/libpref/init/StaticPrefList.yaml
@@ -10201,6 +10201,8 @@
   value: true
 #elif defined(XP_OPENBSD)
   value: true
+#elif defined(XP_SOLARIS)
+  value: true
 #else
   value: false
 #endif
